docs/Users_Guide/roc_diagram.rst

```diff
@@ -82,40 +159,37 @@ the roc_diagram_defaults.yaml configuration file looks like the following:
 .. image:: roc_diagram_default.png
 
 
-To generate the above plot, use the roc_diagram_defaults.yaml and
-the empty custom configuration file, minimal_roc_diagram.yaml.
-Then, perform the following:
+Perform the following:
 
-* verify that you are running in the conda environment that
+* if using a conda environment, verify that you are running in the conda environment that
   has the required Python packages outlined in the requirements
   section
 
 * cd to the $METPLOTPY_SOURCE/METplotpy/metplotpy/plots/roc_diagram
```

Comment:
Instead of suggesting the user "cd to the $METPLOTPY_SOURCE/METplotpy/metplotpy/plots/roc_diagram directory" I think it would be better to have them run from their working directory, and instead specify the full path to the .py file, that way if they don't want to, they don't have to update the plot_filename path and are less likely to run into any permissions problems.  So, for example, instead of running the following:
```
python roc_diagram.py $WORKING_DIR/minimal_roc_diagram.yaml
```
they would run the following:
```
python $METPLOTPY_SOURCE/METplotpy/metplotpy/plots/roc_diagram/roc_diagram.py $WORKING_DIR/minimal_roc_diagram.yaml
```